### PR TITLE
Added Syntax Highlighting and corrected a typo.

### PR DESCRIPTION
**Pull request by fernandosoto138**
# Added Syntax Highlighting

## Summary

Added C# after two ```  and corrected a typo.

### CODE REVIEW

File: docs/csharp/language-reference/keywords/abstract.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The `abstract` modifier indicates that the thing being modified has a missing or
   
 -   Because an abstract method declaration provides no actual implementation, there is no method body; the method declaration simply ends with a semicolon and there are no curly braces ({ }) following the signature. For example:  
   
-    ```  
+    ```C#
```
Comment (JRAlexander) [inline docs/csharp/language-reference/keywords/abstract.md]: Please change C# to csharp.

File: docs/csharp/language-reference/keywords/abstract.md [hunk 2]
```diff
@@ -72,11 +72,11 @@ The `abstract` modifier indicates that the thing being modified has a missing or
   
  In the preceding example, if you attempt to instantiate the abstract class by using a statement like this:  
   
-```  
+```C#
```
Comment (JRAlexander) [inline docs/csharp/language-reference/keywords/abstract.md]: Please change C#  to csharp.

**Review by JRAlexander — CHANGES_REQUESTED** [review]
Thanks, @fernandosoto138, for your contribution! Just a couple of small changes.

File: docs/csharp/language-reference/keywords/abstract.md (hunk 1 above)
Comment (fernandosoto138) [reply docs/csharp/language-reference/keywords/abstract.md]: Done!

**Comment by JRAlexander** [comment]
Thanks for revising, @fernandosoto138 ! We'll merge your PR as soon as the internal build completes. 
It should be live on docs.microsoft.com in a day or two.

**Review by JRAlexander — APPROVED** [review]

**Pull request merged**